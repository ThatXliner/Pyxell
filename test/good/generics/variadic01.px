
func product<T>(T... xs) T def
    return xs.reduce(_*_)

print product(1)
print product(3/4, 3)
print product(2, 5.5, 3/2)
