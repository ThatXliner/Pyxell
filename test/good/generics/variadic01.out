1
9/4
16.5
